tests/sympc/tensor/mpc_tensor_test.py

```diff
@@ -787,3 +787,17 @@ def test_session_ring_xor(get_clients, security, bit) -> None:
     result = operator.xor(x, b)
     expected_res = secret_x ^ secret_b
     assert (result.reconstruct(decode=False) == expected_res).all()
+
+
+def test_mpctensor_reciprocal(get_clients):
```

Comment:
MAYBE: Could you leave only `test_reciprocal` no need to specify mpc tensor since it is in the `mpc tensor file`

Reply:
Done